features/ccx-notification-service/customer_notifications.feature

```diff
@@ -46,7 +46,7 @@ Feature: Customer Notifications
      When I select all rows from table reported
      Then I should get 1 rows
 
-  Scenario: Check that notification service produces instant notifications multiple events same cluster
+  Scenario: Check that notification service produces instant notification single event for same cluster
```

Comment:
Please change to "Check that notification service produces a single notification event for cluster with multiple new reports"

Reply:
Changed :)